cg/services/orders/validation/errors/case_sample_errors.py

```diff
@@ -142,6 +142,11 @@ class SexSubjectIdError(CaseSampleError):
     message: str = "Another sample with the same subject id has a different sex"
 
 
+class CaptureKitResetError(CaseSampleError):
+    field: str = "warnings"
+    message: str = "Capture Kit was reset since it will not be used with this application."
```

Comment:
The rest of the messages don't end with point
```suggestion
    message: str = "Capture Kit was reset since it will not be used with this application"
```